spec/features/create_profile_spec.rb

```diff
@@ -9,6 +9,7 @@
     visit root_path
     click_link "slack-login"
     click_on("Select Graduation year")
+    expect(page).to_not have_content("2030")
```

Comment:
I'm realizing maybe this should be a more flexible expectation because eventually it will be the year 2030... I'm going to update this test